### ISSUE TITLE
Add well connection factors as HM parameter

### ISSUE THREAD

**Issue by edubarrosTNO** [equinor/flownet#419]
During our FlowNet discussions this week, we saw examples by @tayloris where estimating well connection factors through HM (besides the other tube flow properties) helped to better constrain BHP response of flow network models. Given recent difficulties of matching pressures in the Norne case, enabling the estimation of well connection factors within the FlowNet framework is expected to contribute to an improved match

**Comment by tayloris** [comment]
Yes, the well productivity index WI has a direct impact on good BHP prediction. 
So I think one the first task would be to create a WI class, similar to _porv_poro_trans.py  to do the rendering of the parameter values.

**Comment by tayloris** [comment]
I think that the sections `WELSPECS` and `COMPDAT`  from `HISTORY_SCHEDULE` should be indluded in a separate file `SCHEDULE` to facilitate the rendering of the `CONFACT` parameters.

**Comment by edubarrosTNO** [comment]
I completely agree with you, @tayloris . I made the same suggestion in the past, to split SCHEDULE and COMPDAT files but the issue didn't get prioritized - back then it was just to improve readability of the generated SCHEDULE file, but now it looks like it would help code-wise.

A question that comes up then is how to approach this estimation of well connection factor parameters when we have multiple instances of the COMPDAT keyword for the same well in the schedule, i.e. workover in time throughout the simulation. Any thoughts here? I guess that in your Egg model example the well connections were kept fixed through the simulation, right?

**Comment by tayloris** [comment]
Yes, in the case of workover you increase the numbers of connection factor parameters by the number of workovers. 
This can make it complicated because we may need to render COMPDAT and WELLSPEC at each time step. Does Norne have workovers in some wells?
